apps/src/blockly/addons/cdoXml.js

```diff
@@ -153,7 +154,11 @@ export function addMutationToBehaviorDefBlocks(blockElement) {
   // (e.g. shared behaviors).
   // In CDO Blockly, the 'usercreated' flag was set on the block. Google Blockly
   // expects this kind of extra state in a mutator.
-  const userCreated = readBooleanAttribute(blockElement, 'usercreated');
+  const userCreated = readBooleanAttribute(
+    blockElement,
+    'usercreated',
```

Comment:
would it be worth having constants for `usercreated` and `userCreated` (maybe named `userCreatedXML` and `userCreatedJSON` so we don't risk messing up the capitalization?

Reply:
I like this idea and started implementing this change, and then I wondered if it makes the code less readable once you start using `userCreatedJson` all the places we use ``userCreated`` (do we reference `state[userCreatedJson]` in the mutator?). What do you think of creating a userCreatedXml constant to highlight when we're directly reading it from the start blocks and leaving the camelCase use cases untouched?

Reply:
that seems reasonable! We could still use a constant in place of the hard-coded string, to avoid typos.